src/ebay-tabs/__tests__/index.spec.tsx

```diff
@@ -3,23 +3,44 @@ import { render, fireEvent } from '@testing-library/react'
 import { initStoryshots } from '../../../config/jest/storyshots'
 import { EbayTabs, EbayTab, EbayTabPanel } from '../index'
 
+const spy = jest.fn()
```

Comment:
Careful with moving the spy or mock to outside the test scope, especially for a shared spy among tests. You need to reset its status on every test, either by:

- call `spy.mockClear()` on `beforeEach()`
- or re-create the mock on `beforeEach()` (e.g `spy = jest.fn()`) since people might get confused when to call `mockClear()` vs `mockReset()`

Otherwise, if someone add another test with assertion `expect(spy).toBeCalled()`, that assertion will always pass regardless the implementation.